glean-core/android/src/test/java/mozilla/telemetry/glean/acmigration/GleanDataMigrationTest.kt

```diff
@@ -60,6 +61,19 @@ class GleanDataMigrationTest {
         prefs?.edit()?.putInt("${pingName}_seq", number)?.apply()
     }
 
+    private fun waitForSpecificPing(pingServer: MockWebServer, pingName: String): JSONObject {
+        do {
+            // Get the pending pings from the ping server.
+            val request = pingServer.takeRequest(20L, TimeUnit.SECONDS)
+
+            // Check that we received the expected sequence number for the baseline ping.
+            val baselineJson = JSONObject(request.body.readUtf8())
```

Comment:
`baselineJson` -> `pingJson` (since `pingName` could theoretically be anything)...